src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -356,6 +357,22 @@ public function write($messages, $newline = false, $type = self::OUTPUT_NORMAL)
     }
 
     /**
+     * @return QuestionHelper|null
```

Comment:
I don't feel like creating a `SymfonyQuestionHelper` object in the constructor is a good idea. There are many use cases where it's not used at all. Still, `getQuestionHelper` use case is very unclear for me, could you explain your point of view a little bit more please?